client/my-sites/site-settings/site-tools/index.jsx

```diff
@@ -33,7 +33,21 @@ class SiteTools extends Component {
 	}
 
 	render() {
-		const { translate } = this.props;
+		const {
+			translate,
+			site: { jetpack: isJetpack, is_vip: isVip },
+		} = this.props;
+
+		const showSection = {
```

Comment:
Would it make sense to place that logic in `mapStateToProps`? That could possibly prevent some unnecessary renders.